ballerina/Dependencies.toml

```diff
@@ -55,7 +55,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "observe"
-version = "1.0.2"
+version = "1.0.4"
```

Comment:
Minor version updates of dependencies are not reflected here, If there are changes to minot version. Delete the dependencies toml and run the build

Reply:
The minor version dependency update is related to the `integration-tests` package `Dependencies.toml` file which is not being committed.